source/infrastructure/lib/api/model-schema/deploy-usecase-body.ts

```diff
@@ -358,6 +359,34 @@ export const deployUseCaseBodySchema: JsonSchema = {
             required: ['KnowledgeBaseType'],
             additionalProperties: false
         },
+
+        AuthenticationParams: {
+            type: JsonSchemaType.OBJECT,
+            description: 'Parameters related to the Authentication.',
+            properties: {
+                AuthenticationProvider: {
+                    type: JsonSchemaType.STRING,
+                    description: 'Supported authentication provider.',
+                    enum: SUPPORTED_AUTHENTICATION_PROVIDERS
+                },
+                CognitoParams: {
+                    type: JsonSchemaType.OBJECT,
+                    description: 'Cognito user pool related parameters.',
+                    properties: {
+                        ExistingUserPoolId: {
+                            type: JsonSchemaType.STRING,
+                            description: 'Existing Cognito User Pool Id.'
```

Comment:
Sorry I missed this before, but please add a regex for allowed values (`^$|^[0-9a-zA-Z_-]{9,24}$`)